Automattic/wp-calypso
# Anchor: Remove font pairing step placeholder from Anchor flow

#### Changes proposed in this Pull Request

* Remove references to the font pairing step placeholder from the Anchor.fm signup flow.

#### Testing instructions

* Switch to this PR
* Navigate to `/setup/?siteSlug=yoursite.wordpress.com&anchor_podcast=true`
* You should land on the Anchor.fm podcast title screen
* Clicking Continue brings you to the Anchor design selection step
* Clicking on a theme brings you to the editor rather than the font pairing step
* Clicking Back at any point in the flow brings you to the expected place

Related to https://github.com/Automattic/wp-calypso/pull/62750#issuecomment-1108574144
